beetmoverscript/task.py

```diff
@@ -17,6 +19,27 @@ def validate_task_schema(context):
     scriptworker.client.validate_json_schema(context.task, task_schema)
 
 
+def validate_task_scopes(context, manifest):
+    # make sure scopes exist and are properly set
+    scopes = context.task['scopes']
+    for scope in scopes:
+        if scope.startswith("project:releng:beetmover:"):
+            signing_cert_name = scope.split(':')[-1]
+            if re.search('^[0-9A-Za-z_-]+$', signing_cert_name) is not None:
+                break
+            log.warning('scope {} is malformed, skipping!'.format(scope))
+    else:
+        log.critical("No beetmover scopes!")
+        sys.exit(3)
+
+    # prevent uncoordination between the bucket supposed to use for beetmove
+    # and the current credentials scopes
+    if (signing_cert_name not in manifest['s3_prefix_dated'] or
+            signing_cert_name not in manifest['s3_prefix_latest']):
```

Comment:
I would have leaned towards removing the previous check.  One side effect of leaving the previous check is @lundjordan won't be able to test until he changes the in-tree scopes to `project:releng:beetmover:nightly`, since the `s3_prefix`es don't contain `dep`.

At some point, maybe let's change the `s3_prefix`es to be "pub/mobile/{scope}/..." and format.
#28  will also help here.